src/org/loklak/harvester/TwitterScraper.java

```diff
@@ -810,8 +810,7 @@ public static String unshorten(String text) {
             try {
                 Matcher m = timeline_embed_pattern.matcher(text);
```

Comment:
also you need to remove this method's implementation at here ->
https://github.com/Achint08/loklak_server/blob/82357c2671496fee9bd7a1ff8ceea3ed3c26f898/src/org/loklak/harvester/TwitterScraper.java#L723

please also check in other lines, if i missed